src/main/resources/avro/reads.avdl

```diff
@@ -190,7 +190,12 @@ record ReadAlignment {
   /** The fragment is a PCR or optical duplicate (SAM flag 0x400). */
   union { null, boolean } duplicateFragment = null;
 
-  /** The number of reads in the fragment (extension to SAM flag 0x1) */
+  /**
+   The number of reads in the fragment.
+   A value of 1 means SAM flag 0x1 is unset.
```

Comment:
Technically the mapping to SAM would be: 

```
1 -> 0x1 unset
[2, infinity) -> 0x1 set
```

I would guess that `null` means an unknown number of reads, or it shouldn't be allowed.